apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -313,10 +325,15 @@ export default class ExportHTMLModal extends Vue {
   iconDataToSVG(
     iconData: string,
     fill: string,
+    description = 'None',
```

Comment:
would probably make this a purely an optional parameter instead of one that has a default value (would also probably reorder the parameters so this one goes last in line).  if the user intentionally supplies a description, then we should use that.  otherwise, we can move on when we see that it's undefined.

Reply:
Changed as requested.